src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -44,20 +41,15 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId })
   return (
     <Flex alignItems='center' flexDir={{ base: 'column', lg: 'row' }} flex={1} py={4}>
       <Flex alignItems='center' mr='auto'>
-        <SkeletonCircle boxSize='40px' isLoaded={isLoaded}>
-          <Image src={icon} boxSize='40px' fallback={<SkeletonCircle boxSize='40px' />} />
-        </SkeletonCircle>
+        <Image src={icon} boxSize='40px' fallback={<SkeletonCircle boxSize='40px' />} />
```

Comment:
For clarity I'd chuck the JSX value into an `imageFallback` const before the `return`.

```suggestion
        <Image src={icon} boxSize='40px' fallback={imageFallback} />
```